cli/main.go

```diff
@@ -196,6 +201,10 @@ func main() {
 			"Apply a rio file",
 			appName+" up [OPTIONS]",
 			""),
+		builder.Command(&kill.Kill{},
+			"Kill pods or pods from services",
```

Comment:
```suggestion
			"Kill pods or services",
```
unless you really wanted to specify that